Dockerfile

```diff
@@ -6,19 +6,15 @@ ENV RAILS_ENV development
 
 # https://www.mikeperham.com/2018/04/25/taming-rails-memory-bloat/
 ENV MALLOC_ARENA_MAX 2
+ENV QT_QPA_PLATFORM offscreen
 
 RUN apt-get update -qq && apt-get install -y --no-install-recommends curl
 
 RUN curl -sL https://deb.nodesource.com/setup_12.x | bash -
 
-RUN apt-get update && apt-get install --no-install-recommends -y nodejs git build-essential libpq-dev sqlite3 libsqlite3-dev graphicsmagick \
+RUN apt-get update && apt-get install --no-install-recommends -y phantomjs nodejs git build-essential libpq-dev sqlite3 libsqlite3-dev graphicsmagick \
```

Comment:
so we are using `phantomjs` from the Debian package again? I thought it was crashing

Reply:
it was crashing before I added that ENV var. Now it works fine.